client/src/components/header/Header.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+import Box from '@mui/material/Box';
+import styled from '@emotion/styled';
+import { DatePicker, LocalizationProvider } from '@mui/x-date-pickers';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { Store } from '../../utils/store';
+import { Typography } from '@mui/material';
+import Budget from '../../APIs/budget';
+import AddTransaction from './TransactoinModel';
+
+function Navbar() {
+  const store = Store.useContainer();
+
+  const month = store.month.getMonth();
+  const year = store.month.getFullYear();
+
+  const monthName = [
```

Comment:
You can add a Constants File to your project and put all constants there.  https://medium.com/@austinpaley32/how-to-add-a-constants-file-to-your-react-project-6ce31c015774

Reply:
moved constants to separate file in commit 73e8a85af84ec125088f272f3b35e2c3455cb3d4